packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -69,6 +69,15 @@ export const DevServerMixin = {
         return options.strictSSL
     },
 
+    /**
+     * @private
+     */
+    _addEventContext(app) {
```

Comment:
I added an overridden function here since we don't awsServerless middle on local development.